dotcom-rendering/src/web/components/LiveBlock.tsx

```diff
@@ -10,9 +9,7 @@ import { decidePalette } from '@root/src/web/lib/decidePalette';
 import { Hide } from '@root/src/web/components/Hide';
 import { ShareIcons } from '@root/src/web/components/ShareIcons';
 import { headline, textSans } from '@guardian/src-foundations/typography';
-
-const ASIDE_WIDTH = 58;
-const GUTTER = space[3];
+import LiveBlockContainer from '@guardian/common-rendering/src/components/liveBlockContainer';
```

Comment:
Outside this PR but `LiveBlockContainer` is a React component so must be named using a leading capital letter. So the filename should follow this?

Reply:
The convention that's (so far) been established in common-rendering is that the file names of the shared components are lowercased (and in apps-rendering too by the looks of things). Happy to revisit it if this an important question.

Reply:
Shall we review where we are with all these areas that we diverge on? I think it would be great to get them out in the open and make some decisions

Reply:
For sure. Let's talk about it at the next retro 👍🏻